controllers/admin/AdminImportController.php

```diff
@@ -4078,7 +5147,8 @@ protected function supplyOrdersDetailsImportOne($info, &$products, &$reset, $for
 					IFNULL(pa.upc, IFNULL(p.upc, \'\')) as upc
 				');
                 $query->from('product', 'p');
-                $query->leftJoin('product_attribute', 'pa', 'pa.id_product = p.id_product AND id_product_attribute = '.(int)$id_product_attribute);
+                $query->leftJoin('product_attribute', 'pa', 'pa.id_product = p.id_product AND id_product_attribute = '
```

Comment:
If the line is too long, I'd prefer if it was broken up by arguments (one per line).